main/src/ca/uwaterloo/flix/language/phase/unification/EffUnification.scala

```diff
@@ -31,7 +31,9 @@ object EffUnification {
   /**
     * Returns the most general unifier of the two given Boolean formulas `tpe1` and `tpe2`.
     */
-  def unify(tpe1: Type, tpe2: Type, renv0: RigidityEnv)(implicit flix: Flix): Result[(Substitution, List[Ast.BroadEqualityConstraint]), UnificationError] = {
+  def unify(tpe10: Type, tpe20: Type, renv0: RigidityEnv, lenv: LevelEnv)(implicit flix: Flix): Result[(Substitution, List[Ast.BroadEqualityConstraint]), UnificationError] = {
```

Comment:
Why these names tpe10? tpe20?

Reply:
Using the convention that initial values have the `0` suffix. I know you hate it :)

Would you prefer `tpe1_0`?

Or some other idea?

Reply:
Why can it not just be tpe1 and tpe2? :)

Reply:
Because we introduce tpe1 and tpe2 just below.